src/components/Sidebar/Sidebar.jsx

```diff
@@ -294,7 +294,7 @@ const Sidebar = createClass({
 						{barProps.children}
 					</div>
 				</BarPane>
-				<SplitVertical.Divider className={cx('&-Divider')}>
+				<SplitVertical.Divider className={cx('&-Divider')} style={{ width: 7 }}>
```

Comment:
why does this need to be hardcoded in the render function rather than defined in Sidebar.less?

Reply:
because in `SplitVertical`:

```
										style={{
											width: '6px',
											boxSizing: 'border-box',
											...dividerProps.style,
										}}
```

Reply:
The problem is that the jsx styles in SplitVertical would overwrite styles from the Sidebar.less?

Reply:
that's correct

Reply:
The intent of SplitVertical is to style the divider with a default width to be overwritten by a consumer like Sidebar...You could increase the specificity of the selector in less, but it's probably best to descrease the specificity in SplitVertical.less

In any case, this should work for now, but it should be fixed with better css patterns.

Reply:
it's not a selector in less, it's an inline style in jsx. The only way to override it is to pass it in as a style.